src/typechecker.erl

```diff
@@ -73,20 +73,21 @@
              ,tenv             :: #tenv{}
              ,infer    = false :: boolean()
              ,verbose  = false :: boolean()
+             ,exhaust  = true  :: boolean()
              %,tyvenv  = #{}
              }).
 
 %% Two types are compatible if one is a subtype of the other, or both.
 compatible(Ty1, Ty2, TEnv) ->
     case {subtype(Ty1, Ty2, TEnv), subtype(Ty2, Ty1, TEnv)} of
-        {{true, C1}, {true, C2}} ->
-            {true, constraints:combine(C1,C2)};
-        {false, T={true, _C2}} ->
-            T;
-        {T={true, _C1}, false} ->
-            T;
-        {false, false} ->
-            false
+	{{true, C1}, {true, C2}} ->
```

Comment:
Here tabs are introduced in a new place. :-)

Reply:
Oh, so github transforms tags to three spaces. That's why it looks different. 
Also, I thought that I had tabs turned off in my editor but apparently not. I'll make sure to fix that.